src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TornadoService.java

```diff
@@ -260,10 +259,15 @@ public DocumentInfo generateEventDocument(CaseData caseData, String userToken, S
         throws IOException {
         HttpURLConnection connection = null;
         try {
-            dmStoreDocumentName = documentName;
+            String et3DocName = String.format("%s - ET3 Response.pdf",
```

Comment:
is it worth maybe moving this inside the if statement as it will only be used there?

Reply:
done